### PR TITLE
Vehicle data model and trait updates

### PR DESCRIPTION
**Pull request by JFranklin3000**
Closes [issue URL]

##### In this PR
- Restructured the character and common data models to move to where fields were needed. 
##### Testing
- Validate all actors can still be opened.

### CODE REVIEW

File: module/helpers/config.mjs [hunk 1]
```diff
@@ -675,6 +675,77 @@ E20.energonTypes = {
...
+/************************************************
+ * Vehicles                                     *
+ ***********************************************/
+
+E20.vehicleTraits = {
+  aerospace: "VehicleTraitAerospace",
+  ai: "VehicleTraitAI",
+  air: "VehicleTraitAir",
+  allTerrain: "VehicleTraitAllTerrain",
+  ampibious: "VehicleTraitAmphibious",
```
Comment (phildominguez) [inline module/helpers/config.mjs:687]:
```suggestion
  amphibious: "E20.VehicleTraitAmphibious",
```
Needs `E20.` for all of them

File: lang/en.json [hunk 2]
```diff
@@ -581,6 +581,69 @@
...
     "VehiclePassengers": "Passengers",
+    "VehicleTraitAerospace": "Aerospace",
+    "VehicleTraitAI": "A.I.",
+    "VehicleTraitAir": "Air",
+    "VehicleTraitAllTerrain": "All-Terrain",
+    "VehicleTraitAmphibious": "Amphibious",
+    "VehicleTraitArmoredCabin": "Armored Cabin",
+    "VehicleTraitAttackMode": "Attack Mode",
+    "VehicleTraitAutopilot": "Autopilot",
+    "VehicleTraitAutoPilotAdvanced": "Autopilot, Advanced",
```
Comment (phildominguez) [inline lang/en.json:592]:
```suggestion
    "VehicleTraitAutopilotAdvanced": "Autopilot, Advanced",
```

File: module/helpers/config.mjs [hunk 3]
```diff
@@ -675,6 +675,77 @@ E20.energonTypes = {
...
+  armoredCabin: "VehicleTraitArmoredCabin",
+  attackMode: "VehicleTraitAttackMode",
+  autopilot: "VehicleTraitAutopilot",
+  autopilotAdvanced: "VehicleTraitAutoPilotAdvanced",
```
Comment (phildominguez) [inline module/helpers/config.mjs:691]:
```suggestion
  autopilotAdvanced: "VehicleTraitAutopilotAdvanced",
```

File: lang/en.json [hunk 4]
```diff
@@ -581,6 +581,69 @@
...
+    "VehicleTraitFlyBy": "Fly-By",
+    "VehicleTraitFlyingPodium": "Flying Podium",
+    "VehicleTraitFragile": "Fragile",
+    "VehicleTraitGridjump": "Gridjump",
+    "VehicleTraitFreightCarry": "Freight Carry",
+    "VehicleTraitHeavyWinch": "Heavy Winch",
+    "VehicleTraitHeavyWheels": "Heavy Wheels",
+    "VehicleTraitHISSColumn": "H.I.S.S. Column",
+    "VehicleTraitHover": "Hover",
+    "VehicleTraitHydroPortation": "Hydro-portation",
```
Comment (phildominguez) [inline lang/en.json:615]:
```suggestion
    "VehicleTraitHydroPortation": "Hydro-Portation",
```

**Review by phildominguez — CHANGES_REQUESTED** [review]

File: module/helpers/config.mjs (hunk 3 above)
Comment (JFranklin3000) [reply module/helpers/config.mjs]: fixed

File: module/helpers/config.mjs (hunk 1 above)
Comment (JFranklin3000) [reply module/helpers/config.mjs]: fixed

**Review by phildominguez — APPROVED** [review]

**Pull request merged**